### ISSUE TITLE
Lighthouse/Pagespeed should indicate which images were preloaded

### ISSUE THREAD

**Issue by tomasts248** [GoogleChrome/lighthouse#13304]
![image](https://user-images.githubusercontent.com/32283036/139888081-7044e432-aa9c-411e-a8d9-bfab070d9c8d.png)
In order to check whether preloading is working or not.

**Comment by paulirish** [comment]
This is true. Also it isn't obvious in the devtools network panel. However both tools have the per-request data to know if a request was initiated because of a preload.

I kinda think this signal would be more useful in the network panel... wdyt?
